test/xUnit/csharp/test_Subsystem.cs

```diff
@@ -43,8 +43,7 @@ public static void GetSubsystemInfo()
             Assert.Same(ssInfo2, ssInfo);
 
             ReadOnlyCollection<SubsystemInfo> ssInfos = SubsystemManager.GetAllSubsystemInfo();
-            Assert.Single(ssInfos);
-            Assert.Same(ssInfos[0], ssInfo);
```

Comment:
We should keep this test, and also add one for the `ICrossPlatformDsc`. You may want to rename the variable `ssInfo` to `predictorInfo` to make it clear.

Reply:
Updated. Added method `VerifyAllSubsystemInfo` to check the collection.